### DISCUSSION TITLE
How To Get The Modify Array Checkbox "Append To End" State?

### DISCUSSION THREAD

**Discussion by jincongcong** [jeremytammik/RevitLookup#122] [Q&A] [answered]
Snoop LinearArray，How To Get The Array is "Append To End" or not ?

**Comment by jeremytammik** [comment]
I see no such member on the `LinearArray` class, so I see nothing that could possibly be snooped...

**Comment by Nice3point** [comment, marked as answer]
@jincongcong Apparently you mean this checkbox. 
![изображение](https://user-images.githubusercontent.com/20504884/145619438-caf7de5f-6910-4e1b-951f-9d6b7a7fe7f2.png)
This question does not apply to this add-in, since all properties and methods are displayed. "Append To End" not available in the API
![изображение](https://user-images.githubusercontent.com/20504884/145619728-7e455052-c293-4a8c-ad68-0a57abbfb5cd.png)

**Comment by jeremytammik** [comment]
@jincongcong , if this topic is important to you, I suggest you raise it in the Revit API discussion forum.
